docs/organizations/managing-repositories.md

```diff
@@ -1,10 +1,12 @@
 # Managing repositories
 
-To see a list of the repositories that Codacy is monitoring, open the page **Repositories** under your organization.
+Users with the [necessary permissions on your Git provider](roles-and-permissions-for-organizations.md) can **add** the organization repositories to Codacy to start analyzing them. Other members of the organization will then be able to **follow** on Codacy the repositories of their interest.
 
-![Repositories list](images/repositories.png)
+To see a list of the repositories that you follow on Codacy, open the page **Repositories** under your organization. Across the application, Codacy calculates and displays data for the repositories in this list.
```

Comment:
Minor nitpick regarding [in/on the list](https://forum.wordreference.com/threads/in-the-list-or-on-the-list.469925/) (and another Grammarly suggestion):

```suggestion
To see a list of the repositories you follow on Codacy, open the page **Repositories** under your organization. Across the application, Codacy calculates and displays data for the repositories on this list.
```